bin/run.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+app="improtresk-api"
+
+DIR_ROOT="$(dirname "$(readlink -f "$0")")"
+
+cd $DIR_ROOT
+
+source ../../bin/activate
+
+python ../manage.py migrate
```

Comment:
Automatically triggering migrations could be dangerous. We need to be extra careful with this.

I would backup the database just prior to executing migrations (if the state of the database at that point is not backed up by something different). That could save our ass sometime.

Testing the migration of production database on development environment is also good to do prior migrating the production.